entries/c/cloud.baidu.com.json

```diff
@@ -0,0 +1,17 @@
+{
+    "Baidu AI cloud": {
+      "domain": "cloud.baidu.com",
+      "tfa": [
+        "totp",
+        "custom-software"
+      ],
+      "custom-software": [
+        "Baidu AI Cloud APP"
```

Comment:
The documentation page you linked to doesn't mention MFA using Baidu AI Cloud App. Just SMS and TOTP.

Reply:
Yes, you are right! I was confused with custom identify authenticator and custom software before. This section [Bind the Virtual MFA](https://intl.cloud.baidu.com/doc/IAM/s/2jwvyc50i-en#bind-the-virtual-mfa) mentions MFA using Baidu AI Cloud App just like normal authenticator.